src/components/navItems.js

```diff
@@ -1,12 +1,46 @@
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] The linters are there to enforce code quality and should not be disabled. Kindly enable the linters and fix the issues raised.